src/assets/i18n/en.json

```diff
@@ -17,21 +17,71 @@
   },
   "debrief": {
     "showMeTellMeQuestions": {
-      "T01": "Brakes",
-      "T02": "Safety factors while loading",
-      "T03": "Safety factors while securing a load",
-      "T04": "Sufficient tread",
-      "T05": "Engine coolant",
-      "T06": "Head restraint",
-      "S01": "Direction indicators",
-      "S02": "Doors secure",
-      "S03": "Horn",
-      "S04": "Parking brake",
-      "S05": "Windscreen",
-      "S06": "Demist windscreen",
-      "S07": "Rear fog lights",
-      "S08": "Dipped to main beam",
-      "S09": "Brake lights"
+      "B": {
+        "T01": "Brakes",
+        "T02": "Safety factors while loading",
+        "T03": "Safety factors while securing a load",
+        "T04": "Sufficient tread",
+        "T05": "Engine coolant",
+        "T06": "Head restraint",
+        "S01": "Direction indicators",
+        "S02": "Doors secure",
+        "S03": "Horn",
+        "S04": "Parking brake",
+        "S05": "Windscreen",
+        "S06": "Demist windscreen",
+        "S07": "Rear fog lights",
+        "S08": "Dipped to main beam",
+        "S09": "Brake lights"
+      },
+      "B+E": {
+        "T01": "Brakes",
+        "T02": "Safety factors while loading",
+        "T03": "Safety factors while securing a load",
+        "T04": "Sufficient tread",
+        "T05": "Engine coolant",
+        "T06": "Head restraint",
+        "S01": "Direction indicators",
+        "S02": "Doors secure",
+        "S03": "Horn",
+        "S04": "Parking brake",
+        "S05": "Windscreen",
+        "S06": "Demist windscreen",
+        "S07": "Rear fog lights",
+        "S08": "Dipped to main beam",
+        "S09": "Brake lights"
+      },
+      "C": {
```

Comment:
I've asked Matt Murray to add the spreadsheet with the showMe/TellMe questions for all the vocational tests - if you want, you could include the all the questions in this PR or open another one (since there is a ticket for C1, C1+E, C+E subcategories)